.github/ISSUE_TEMPLATE/DOCUMENTATION.yaml

```diff
@@ -0,0 +1,44 @@
+name: Documentation
+description: Report typos/bugs or suggest new content.
+labels: ["documentation"]
+assignees: 'abbycross, beckykd, javabster'
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for reporting!  Please also use the search to see if there are any other relevant issues or pull requests.
+
+  - type: textarea
+    attributes:
+      label: URL to the relevant documentation
+      description: provide a URL and section name/paragraph if applicable
+    validations:
+      required: false
+
+  - type: checkboxes
+    attributes:
+      label: Select all that apply
+      description: Please select all that apply from the following.
+      options:
+        - label: typo
+        - label: code bug
+        - label: add new content
+        - label: out-of-date content
+        - label: it's just wrong
+        - label: other
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: What is the documentation issue/request?
+      description: If reporting a bug, please be as specific as possible (steps to reproduce, a screenshot etc.) If suggesting new content, please give a clear and concise description. What value does the new content bring?
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Reviewers
+      description: (Optional) List Github usernames of anyone who should be tagged as a reviewer (include yourself)
+    validations:
+      required: false
```

Comment:
Please don't include this - we get too much traffic to be encouraging users to ping maintainers on every issue; we already get notified, and a ping jumps that to "top priority" in the GitHub interface.  It's up to you guys whether you want to be automatically assigned, but even assignment is easier because you can remove it again - we can't undo a direct ping.